ego4d/internal/validate.py

```diff
@@ -593,6 +604,11 @@ def validate_university_files(  # noqa :C901
         write.writerow(fields)
         for e in error_message:
             write.writerow([e.uid, e.errorType, e.description])
+    with open("error_summary", "w") as f:
```

Comment:
nit: you can do this in one line with pandas `pd.DataFrame(error_dict).to_csv(summary_file_path)` (feel free to not address)